NEWS

```diff
@@ -221,9 +227,10 @@ Changed Functionality
 - Zeek's CMake scaffolding has received an overhaul for modernizing the build
   system and to make it easier to maintain going forward. Plugins can now use a
   declarative interface for adding all sources, BIFs, etc. in one block instead
-  of using the previous begin/end functions. However, the old functions still
-  exist for backwards compatibility. Since Zeek uses newer CMake features now,
-  plugin authors should also raise their minimum required CMake version to 3.15.
+  of using the previous begin/end functions. While the old plugin functions
+  still exist for backward compatibility, the underlying codebase requires newer
+  CMake features. Plugin authors should raise their minimum required CMake
```

Comment:
Do we need to mention this version requirement again after we already mentioned it in an earlier section?

Reply:
I figured it doesn't hurt — we've previously erred on the side of not mentioning such things enough.